### PR TITLE
Added BoundedPlainSequences messages

### PR DESCRIPTION
**Pull request by MiguelCompany**
Adding two message definitions with sequences of POD types

### CODE REVIEW

File: msg/BoundedSequencesNoStrings.msg [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Bounded sequences of different types
```
Comment (hidmic) [inline msg/BoundedSequencesNoStrings.msg]: @MiguelCompany if the core idea behind adding messages is to have bounded sequence of plain data types, consider making that explicit in the name e.g. `BoundedPlainSequences`, `BoundedPODSequences` or any meaningful name you like.

File: msg/UnboundedSequencesNoStrings.msg [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Unbounded sequences of different types
```
Comment (hidmic) [inline msg/UnboundedSequencesNoStrings.msg]: @MiguelCompany if you have no use case for this message type, please don't add it.

**Review by hidmic — COMMENTED** [review]
Circling back, sorry for the delay!

**Review by hidmic — APPROVED** [review]

**Comment by hidmic** [comment]
CI up to `rmw_fastrtps_cpp`, `rmw_fastrtps_dynamic_cpp`, `rcl_interfaces` and `test_rmw_implementation`:
* Linux [Build Status](https://ci.ros2.org/job/ci_linux/14795/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9544/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12461/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14942/)

**Comment by hidmic** [comment]
MacOS CI, one more time:
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12630/)

**Comment by hidmic** [comment]
CI again as it's been a while:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14975/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9716/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12655/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15139/)

**Comment by hidmic** [comment]
Alright, finally green. Going in!

**Comment by scpeters** [comment]
This is Steve your friendly build farmer here checking in. I don't know if it was this or https://github.com/ros2/rcl_interfaces/pull/123, but the last nightly build shows quite a few failures in the [test_communication](https://index.ros.org/p/test_communication/) package related to `BoundedPlainSequences`

* https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_debug/lastCompletedBuild/testReport/

I'm not sure if this PR's CI builds included the `test_communication` package or not, but it seems to be unhappy at the moment

**Comment by MiguelCompany** [comment]
@scpeters @hidmic It seems `test_communication` was not included in the PR's CI builds.

I see two different issues here:
1. [ros2/rcl_interfaces#123](https://github.com/ros2/rcl_interfaces/pull/123) did not update function [get_test_msg](https://github.com/eProsima/rcl_interfaces/blob/99c047ce40cc6bfe4b53cecbbeec073342a0cbb5/test_msgs/src/test_msgs/message_fixtures.py#L398). ros2/rcl_interfaces#126 should take care of it
2. CPP executables on `test_communication` package should handle `BoundedPlainSequences` as an argument. Will try to do a PR today for that

**Pull request merged**